XRI3TOXRI2MIGRATIONGUIDE.md

```diff
@@ -0,0 +1,86 @@
+
+<img width="1024" alt="InteractorsInputActions" src="https://github.com/ms-RistoRK/MixedRealityToolkit-Unity/assets/84108471/e6bae726-99b4-4683-8333-f08d90c583d3">
+
+### MRTK InputReaders
+
+As you may have noticed already MRTK controller prefabs now have a new child in which we group Input actions for pinch and UI interaction.  These are shown in the next figure:
+
+<img width="614" alt="InputReaders" src="https://github.com/ms-RistoRK/MixedRealityToolkit-Unity/assets/84108471/9668ae05-7bce-4794-8c08-154a38d93dc9">
+
+We added this child + script as a workaround for devices without interaction profiles for hands, however, if your solution does not have a use-case without hands interactions profiles then you can safely map the MRTK's InputReaders' input actions directly in their corresponding Interactor fields.
+
+### TrackedPoseDriver and ModeManagedRoot references
+
+In addition to the InputReaders child you may have also noticed that MRTK interactors now have two new fields: [Tracked Pose Driver](https://docs.unity3d.com/Packages/com.unity.inputsystem@1.8/api/UnityEngine.InputSystem.XR.TrackedPoseDriver.html) and Mode Managed Root, as shown next:
+
+<img width="368" alt="TPDAndMMR" src="https://github.com/ms-RistoRK/MixedRealityToolkit-Unity/assets/84108471/b1f4e4a6-daff-48d9-9eeb-68e4546070a3">
+
+These are just convenient fields to hold references to the parent controller [Tracked Pose Driver](https://docs.unity3d.com/Packages/com.unity.inputsystem@1.8/api/UnityEngine.InputSystem.XR.TrackedPoseDriver.html) component and GameObject.  These are not mandatory for a successful XRI3 migration but they facilitate coding as well as writing Unity-tests.
+
+## Step 4 - Update scripts
+
+For the XRI3 migration we modified our scripts with two goals:
+
+* Adhere to the new XRI3 patterns.
+* Provide backward compatibility.
+
+As you will notice in the code many of our changes query the Input Compatibility Mode field to differentiate between XRI2 functionality and XRI3 functionality.  This is not mandatory for a successful XRI3 migration, however, we wanted to create a smooth transition for all of you and did our best to ensure backward compatibily as you migrate your forks or solutions.
+
+We encourage you to check the code in each script as well as the commit history to get a sense of the changes that were needed.
+
+## Step 5 - Update Unity-tests
+
+Last but not least, Unity-tests!.  We created several new Unity-tests to ensure the correct functionality of the new XRI3 MRTK rig as well as the old functionality.  We encourage you to visit the code and commit history related to Unity-tests to get a sense of the changes.  Note that the changes focus mostly on where and how the input actions are queried but the overal functionality test remains the same.
```

Comment:
One punctuation mark, please :)

Reply:
Thx for feedback, implemented in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/839/commits/e14267bbcb572fa95f69535cbc5fc9cbb0a22d00, resolving.